wasm/js/skia_renderer.js

```diff
@@ -261,7 +261,9 @@ Module.onRuntimeInitialized = function () {
             _animationCallbackHandler.cancelAnimationFrame.bind(_animationCallbackHandler);
     Rive['enableFPSCounter'] =
             _animationCallbackHandler.enableFPSCounter.bind(_animationCallbackHandler);
-    _animationCallbackHandler.onAfterCallbacks = flushOffscreenRenderers;
+    if (_offscreenGL) {
```

Comment:
Instead of doing the check here, can we instead do:

```
if (!_offscreenGL) {
    return;
}
```

From flushOffscreenRenderers?

I"m thinking about the case where the user adds a offscreen WebGL after the fact, after this code has executed. In that case I think we wouldn't get the flushOffscreenRenderers hook that we need.

Reply:
Also, thanks for catching this and taking care of it!

Reply:
np! Pushed up that change